backend/src/main/kotlin/org/darkSolace/muse/storyModule/model/StoryTag.kt

```diff
@@ -0,0 +1,27 @@
+package org.darkSolace.muse.storyModule.model
+
+import org.hibernate.Hibernate
+import javax.persistence.Entity
+import javax.persistence.Id
+
+@Entity
+data class StoryTag (
+    @Id
+    var id: Long? = null,
+    val tagName: String
+) {
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (other == null || Hibernate.getClass(this) != Hibernate.getClass(other)) return false
+        other as StoryTag
+
+        return id != null && id == other.id
+    }
+
+    override fun hashCode(): Int = javaClass.hashCode()
+
+    @Override
+    override fun toString(): String {
+        return this::class.simpleName + "(id = $id , tagName = $tagName )"
```

Comment:
please tell me it's not yet another convention to have spaces before **and** after commas and to have a space before a closing round bracket but not after an opening one.

Reply:
As this is just an auto-generated string representation of the class my guess would be to prevent multiple "," appearing if an attribute ends in an ",". Changed it anyway 😄